src/test/policyestimator_tests.cpp

```diff
@@ -61,10 +55,10 @@ BOOST_AUTO_TEST_CASE(BlockPolicyEstimates)
     // This makes the tx count about 1.33 per bucket, above the 1 threshold
     while (blocknum < 200) {
         for (int j = 0; j < 10; j++) { // For each fee/pri multiple
-            for (int k = 0; k < 5; k++) { // add 4 fee txs for every priority tx
+            for (int k = 0; k < 5; k++) { // add 4 fee txs  
```

Comment:
Trailing whitespace.

Reply:
i should have changed the range to `for (int k = 0; k < 4; k++)` here. this is because `k` is used to index `feeV[k/4][j]`. `feeV[0]` was used to track fee transactions, and `feeV[1]` was used to track priority transactions.